BlogTemplate/Models/Post.cs

```diff
@@ -10,10 +11,13 @@ namespace BlogTemplate.Models
     public class Post
     {
         public List<Comment> Comments { get; set; } = new List<Comment>();
+        [Required(ErrorMessage = "Title required")]
         public string Title { get; set; }
+        [Required(ErrorMessage = "Post text required")]
         public string Body { get; set; }
         public DateTime PubDate { get; set; } = DateTime.Now;
         public DateTime LastModified { get; set; } = DateTime.Now;
+        public string TagsString { get; set; }
```

Comment:
Let's see if we can bind with Tags better.  Having a duplicated property for the string vs. the list means we have to keep them in sync.